apps/evm/src/ui/pool/Steer/SteerLiquidityManagement/Remove/SteerPositionRemove.tsx

```diff
@@ -62,7 +63,8 @@ export const SteerPositionRemove: FC<SteerPositionRemoveProps> = ({
   const slippagePercent = useMemo(() => {
     return new Percent(
       Math.floor(
-        +(slippageTolerance === 'AUTO' ? '0.1' : slippageTolerance) * 100,
+        +(slippageTolerance === 'AUTO' ? DEFAULT_SLIPPAGE : slippageTolerance) *
```

Comment:
Still seems weird to be repeating this here and newing up new percent object in each of these views.